core/src/listener/state.ts

```diff
@@ -252,10 +242,10 @@ export class State {
     )
 
     if (jobs.length != 1) {
-      throw new OraklError(
-        OraklErrorCode.UnexpectedNumberOfJobsInQueue,
-        `Number of jobs ${jobs.length}`
-      )
+      // throw new OraklError(
+      //   OraklErrorCode.UnexpectedNumberOfJobsInQueue,
+      //   `Number of jobs ${jobs.length}`
+      // )
```

Comment:
Is there specific reason for this being commented?

Reply:
ah that error was being thrown locally so I had to comment it out, I'll push a commit uncommenting it. Thanks for catching it 😄